packages/app/src/messages/components/messages-input-search/messages-input-search.interface.ts

```diff
@@ -3,11 +3,11 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-import { Thread } from "App/messages/dto"
+import { Thread, Message } from "App/messages/dto"
 import { SearchResult } from "App/search/dto"
 
 export interface MessagesInputSearchProps {
-  onSelect: (thread: Thread) => void
+  onSelect: (record: Thread & Message) => void
```

Comment:
Would be good to apply the same as in https://github.com/mudita/mudita-center/pull/945#discussion_r958503690

Reply:
https://github.com/mudita/mudita-center/pull/945/commits/735a58b6787a26a9f9fc3714fa5138d4dc264f41